armsrc/iclass.c

```diff
@@ -2152,3 +2152,216 @@ void iClass_Restore(iclass_restore_req_t *msg) {
         reply_ng(CMD_HF_ICLASS_RESTORE, isOK, NULL, 0);
     }
 }
+
+void generate_single_key_block_inverted(const uint8_t startingKey[PICOPASS_BLOCK_SIZE], uint32_t index, uint8_t keyBlock[PICOPASS_BLOCK_SIZE]) {
```

Comment:
use `const uint8_t *starting_key`   instead of arrays with defined sizes.

Reply:
Fixed